---
bool done
float32 reward
bool wrong_altitude
float32 x
float32 y
float32 z
float32 orientation_x
float32 orientation_y
float32 orientation_z
float32 orientation_w
